### ISSUE TITLE
MBartForConditionalGeneration to do mask filling task with mbart-large-50-many-to-many-mmt

### ISSUE THREAD

**Issue by Aureole-1210** [huggingface/transformers#28990]
### System Info

I also have problem with this.
I want to use 【facebook/mbart-large-50-many-to-many-mmt】 to do mask filling task. But the output is always strange.
I modify the input format as the Model Card from https://huggingface.co/facebook/mbart-large-50-many-to-many-mmt suggested.
My code is as follows:

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

```python
from transformers import (
AutoTokenizer,
BertForMaskedLM,
MBart50TokenizerFast,
MBartForConditionalGeneration,
DataCollatorForLanguageModeling
)
model_name_or_path = 'my_path/mbart-large-50-many-to-many-mmt'
model = MBartForConditionalGeneration.from_pretrained(model_name_or_path)
tokenizer = MBart50TokenizerFast.from_pretrained(model_name_or_path)

tokenizer.src_lang = 'en_XX'

src = "So that such a thing won’t happen <mask>."
encoded_src = tokenizer([src], return_tensors="pt")
input_ids = encoded_src["input_ids"]
src_tokens = tokenizer.convert_ids_to_tokens(input_ids[0])

model_outputs = model(**encoded_src)
logits = model_outputs.logits

masked_index = torch.nonzero((input_ids[0] == tokenizer.mask_token_id)).item()
probs = logits[0, masked_index].softmax(dim=0)
values, predictions = probs.topk(5)

print(tokenizer.convert_ids_to_tokens(predictions))
```

The output is:
['.', '☎', '↔', '∏', '∴']

### Expected behavior

When I change my input, it always output strange symbols. I think this is wrong.
I think the output should be English words at least.

I am confused whether this model is not suitable for this task. How should I modify to get proper outputs? Thank you so much!

**Comment by amyeroberts** [comment]
cc @ArthurZucker

**Comment by ArthurZucker** [comment]
This is a duplicate of #25425, you are using `'my_path/mbart-large-50-many-to-many-mmt'` which I cannot access. Could you try with an official checkpoint and read the linked issue? 🤗
